### PR TITLE
Fix a typo in the spec (and make corresponding change in node stress client)

### PR DESCRIPTION
**Pull request by sreecha**
s/test_duration-secs/test_duration_secs

### CODE REVIEW

File: src/node/stress/stress_client.js [hunk 1]
```diff
@@ -118,8 +118,8 @@ function main() {
   }));
   start(server_addresses, test_cases, argv.num_channels_per_server,
         argv.num_stubs_per_channel, argv.metrics_port);
-  if (argv['test_duration-secs'] > -1) {
-    setTimeout(stop, argv['test_duration-secs'] * 1000);
+  if (argv['test_duration_secs'] > -1) {
+    setTimeout(stop, argv['test_duration_secs'] * 1000);
```
Comment (murgatroid99) [inline src/node/stress/stress_client.js]: These two lines should now use `argv.test_duration_secs`

Comment (sreecha) [inline src/node/stress/stress_client.js]: Done

**Comment by murgatroid99** [comment]
LGTM

**Comment by sreecha** [comment]
All tests passed (the failure in basic tests in a known python flake). Adding ready to merge label.

**Pull request merged**